internal/rules/policies/cloud/policies/aws/rds/enable_deletion_protection.rego

```diff
@@ -0,0 +1,25 @@
+# METADATA
+# title: "RDS Deletion Protection Disabled"
+# description: "Ensure deletion protection is enabled for RDS database instances."
+# scope: package
+# schemas:
+# - input: schema.input
+# related_resources:
+# - https://aws.amazon.com/about-aws/whats-new/2018/09/amazon-rds-now-provides-database-deletion-protection/
+# custom:
+#   avd_id: AVD-AWS-0177
```

Comment:
You can find the AVD ID to use by running `make id` and picking the id from the relevant provider (e.g. `AWS` in this case.